### ISSUE TITLE
Facet by language code(s)

### ISSUE THREAD

**Issue by thisisaaronland** [whosonfirst/whosonfirst-www-spelunker#105]
Specifically we want the ability filter for things with language `x` but also the ability to filter for things _without_ language `x`.

So: We should keep a hard-coded list of languages that we want to "negate" by and then just add them, where appropriate, as UI elements (rather than ES results data).

This is blocked on:

* https://github.com/whosonfirst/whosonfirst-www-spelunker/issues/106

I don't think we need to do any new or special indexing hoohah in: https://github.com/whosonfirst/py-mapzen-whosonfirst-search/blob/master/mapzen/whosonfirst/search/__init__.py

**Comment by thisisaaronland** [comment]
This is actually also blocked on:

https://github.com/whosonfirst/py-mapzen-whosonfirst-search/issues/26

**Comment by nvkelso** [comment]
Super cool, the == is working for me on prod :)

But =! (errr, !=) doesn't seem to work, I get a 500 error? I tried English, German, and a few others.

https://whosonfirst.mapzen.com/spelunker/id/102191575/descendants/?exclude=nullisland&translations=deu&translations=!eng

<img width="1191" alt="screen shot 2017-07-07 at 14 15 44" src="https://user-images.githubusercontent.com/853051/27977273-cf0e693a-631e-11e7-98bb-ce964404ece4.png">

**Comment by thisisaaronland** [comment]
It's not supporting multiple translations correctly – I will file a separate ticket:

https://whosonfirst.dev.mapzen.com/spelunker/id/102191575/descendants/?exclude=nullisland&translations=!eng <-- works

**Comment by thisisaaronland** [comment]
Fixed.

**Comment by nvkelso** [comment]
Rad! (Didn't realize it was 2 levels of language deep!)
